src/main/java/com/musala/sdcs/devices/MotionSensorBuilder.java

```diff
@@ -0,0 +1,33 @@
+package com.musala.sdcs.devices;
+
+public class MotionSensorBuilder {
+    
+    private String manufacturer = "";
```

Comment:
We have to discuss if we need to provide default values to all fields in the builder. :)